storage/container_storage_interface/persistent-storage-csi-vsphere.adoc

```diff
@@ -0,0 +1,33 @@
+[id="persistent-storage-vsphere"]
+= VMware vSphere CSI Driver Operator
+include::modules/common-attributes.adoc[]
+:context: persistent-storage-csi-vsphere
+
+toc::[]
+
+== Overview
+
+{product-title} can provision persistent volumes (PVs) using the Container Storage Interface (CSI) VMware vSphere driver for Virtual Machine Disk (VMDK) volumes.
+
+:FeatureName: vSphere CSI Driver Operator
+include::modules/technology-preview.adoc[leveloffset=+1]
+
+Familiarity with xref:../../storage/understanding-persistent-storage.adoc#understanding-persistent-storage[persistent storage] and xref:../../storage/container_storage_interface/persistent-storage-csi.adoc#persistent-storage-csi[configuring CSI volumes] is recommended when working with a CSI Operator and driver.
+
+To create CSI-provisioned persistent volumes (PVs) that mount to vSphere storage assets, {product-title} installs the vSphere CSI Driver Operator and the vSphere CSI driver by default in the `openshift-cluster-csi-drivers` namespace.
+
+* *vSphere CSI Driver Operator*: By default, the Operator provides a storage class (`thin-csi`) that you can use to create persistent volumes claims (PVCs). The vSphere CSI Driver Operator supports dynamic volume provisioning by allowing storage volumes to be created on-demand, eliminating the need for cluster administrators to pre-provision storage.
+
+* *vSphere CSI driver*: The driver enables you to create and mount vSphere PVs.
+
+[IMPORTANT]
+====
+{product-title} defaults to using an in-tree (non-CSI) plug-in to provision vSphere storage. This in-tree plug-in will be removed in a subsequent update of {product-title}. Volumes provisioned using the existing in-tree plug-in are planned for migration to the CSI driver at that time.
+====
```

Comment:
Do we really want the wording about intree driver removal? I worry if this will scare some people. I think that, we should be careful here and may be document eventual removal of in-tree driver when we can GA the CSI driver.

Reply:
I think we could reword the sentence to be less scary. Like the CSI migration should be seamless and existing APi objects (PVs, PVCs, StorageClasses) can be still used. The migration should be a technical detail that nobody notices (if we do it right).

Reply:
This is boilerplate text that was being added to all of the new CSI drivers that have an equivalent in-tree driver. If we need to revise it here should it be revised globally? Should it just be deleted? Let's discuss in next stand up mtg.